### DISCUSSION TITLE
绳网改包名！！！

### DISCUSSION THREAD

**Discussion by duoduo13123678** [share121/inter-knot#4548] [科技]
绳网包名居然只是com.example.inter_knot，有example怎么可以?！必须改成com.share121.inter_knot！！！自己应用加自己的元素没问题吧？
![Screenshot_2024-09-07-19-43-40-017_com miui securitycenter](https://github.com/user-attachments/assets/62e0ae68-72f5-4f3f-b4d9-9c5e1db44c10)

**Comment by share121** [comment]
已解决

**Reply by shidijun** [reply]
抓到了awa
